translations.schema.json

```diff
@@ -181,6 +181,9 @@
             "volume": {
               "$ref": "#/definitions/translation_name"
             }
+            "interval": {
```

Comment:
it looks this should be "clean_pads_interval" instead of "interval".
Also in line 183, a comma is missing.